### PR TITLE
Created Products Page

### PR DESCRIPTION
**Pull request by pmq24**
Reformated code conventions in .tsx files, added package-lock.json to .gitignore

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -21,3 +21,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+package-lock.json
```
Comment (pmq24) [inline .gitignore]: `package-lock.json` MUST NOT be ignored

File: src/App.tsx [hunk 2]
```diff
@@ -1,28 +1,10 @@
-import React, { useState, useEffect } from 'react';
-import { productProps } from './components/models';
-import './components/ProductCard'
-import ProductList from './components/ProductList';
+import React from 'react';
+import ProductsPage from './pages/ProductsPage';
 
 function App() {
```
Comment (pmq24) [inline src/App.tsx]: This line, with line 10, can be inlined-ly combined to `export default function App() {`

File: src/components/ProductCard.tsx [hunk 3]
```diff
@@ -7,7 +7,7 @@ import { CardActionArea } from '@mui/material';
 import { productProps } from './models';
 import './style.css'
 
-const ProductCard = (product: productProps) => {
+export default function ProductCard(product: productProps) {
```
Comment (pmq24) [inline src/components/ProductCard.tsx]: The properties of React components should be named `props`.

If you name it as `product`, line 13 would be `key={product.product.id}`

**Review by pmq24 — COMMENTED** [review]
@MinhKhang0715 You pick up React very quickly! Very nice!

Just need some minor issues, please re-request review when you're done fixing them!

File: src/components/ProductCard.tsx (hunk 3 above)
Comment (MinhKhang0715) [reply src/components/ProductCard.tsx]: There is no property named `product` in the `productProps` interface though. So `product.product.id` would not make sense

**Pull request merged**